bin/rails

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby.exe
```

Comment:
- [ ] [OPTIONAL] Kindly remove the `.exe.` from this file so users using macOS or Linux can run your app, you can replace this string with `#!/usr/bin/env ruby`